src/qml/QFieldSettings.qml

```diff
@@ -64,49 +64,339 @@ Page {
+                    border.color: parent.parent.checked ? Theme.mainColor : Theme.lightGray
+                    Behavior on x {
+                        PropertyAnimation {
+                            duration: 150
+                            easing.type: Easing.Linear
+                        }
+                    }
+                }
+            }
+          }
 
-        Switch {
-          id: incrementalRenderingCheckBox
-          width: parent.width
-          text: qsTr( "Redraw map every 250 ms while rendering" )
-        }
+          SwitchDelegate {
+              id: fullScreenIdentifyViewCheckBox
+              width: parent.width
+              padding: 25 * dp
+              topPadding: 28 * dp
+              text: qsTr( "Maximized attribute form" )
+              contentItem: Text {
+                  text: parent.text
+                  font: Theme.defaultFont
+                  verticalAlignment: Text.AlignVCenter
+                  wrapMode: Text.WordWrap
+                  Layout.fillWidth: true
+                  Layout.fillHeight: true
+                  Layout.minimumHeight: contentHeight
+                  rightPadding: parent.indicator.width + parent.spacing
+              }
+              background: Rectangle {
+                  implicitWidth: parent.width
+                  color: transparent
+              }
+              indicator: Rectangle {
+                  implicitWidth: 48 * dp
+                  implicitHeight: 26 * dp
+                  x: parent.width - width - parent.rightPadding
+                  y: ( parent.height + parent.topPadding - 4 * dp ) / 2 - height / 2
+                  radius: 13 * dp
+                  color: parent.checked ? Theme.mainColor : Theme.lightGray
+                  border.color: parent.checked ? Theme.mainColor : Theme.lightGray
 
+                  Rectangle {
+                      x: parent.parent.checked ? parent.width - width : 0
+                      width: 26 * dp
+                      height: 26 * dp
+                      radius: 13 * dp
+                      color: parent.parent.down ? Theme.lightGray : Theme.light
+                      border.color: parent.parent.checked ? Theme.mainColor : Theme.lightGray
+                      Behavior on x {
+                          PropertyAnimation {
+                              duration: 150
+                              easing.type: Easing.Linear
+                          }
+                      }
+                  }
+              }
+          }
 
-        Switch {
-          id: numericalDigitizingInformationCheckBox
-          width: parent.width
-          text: qsTr( "Show numerical information while digitizing" )
-          checked: true
-        }
+          SwitchDelegate {
+              id: locatorKeepScaleCheckBox
+              width: parent.width
+              padding: 25 * dp
+              topPadding: 28 * dp
+              text: qsTr( "Fixed scale navigation" )
```

Comment:
Can you add a description that includes "locator" here?

Reply:
'When fixed scale navigation is active, focusing on a located feature will pan but not zoom onto the feature location' ?